app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -13,6 +13,7 @@ def user_info
       sub: uuid_from_sp_identity(identity),
       iss: root_url,
       email: email_from_sp_identity(identity),
```

Comment:
This reuses the existing email scope, what if we added a new scope so this attribute becomes opt in?

bonus, as a separate scope it would also trigger the permission screen again right?